src/pychart/runner.py

```diff
@@ -22,13 +22,17 @@ def run(source: str):
 
 
 def run_prompt():
-    while True:
-        line = input("$ : ")
+    try:
+        while True:
+            line = input("$ : ")
 
-        if line == ".exit":
-            break
+            if line == ".exit":
```

Comment:
I think when we add statements we should add an `exit()` function or something then we can git rid of this